templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+ <h1>Delete driver</h1>
+
+  <p>Are you sure to delete driver: {{ driver }}?</p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-outline-danger"/>
```

Comment:
not obligatory but it would be more comfortable for user if this form had button "Cancel" or "No"